ComponentKitTests/CKComponentGestureActionsTests.mm

```diff
@@ -138,6 +138,16 @@ - (void)testThatWithNoDelegateActionsNoDelegateIsSet
   XCTAssertNil(gesture.delegate, @"Gesture delegate should not be set");
 }
 
+- (void)testThatApplyingANilRecognizerClassResultsInNoRecognizer
+{
+  CKComponentViewAttributeValue attr = CKComponentGestureAttribute(Nil, nullptr, @selector(test));
```

Comment:
Fair. Let's stop the bleeding — if you put out a PR to change it to a `CKAssert` I'd stamp that.